diff --git a/configuration.py b/configuration.py
index 5a6d978..4d0a316 100644
--- a/configuration.py
+++ b/configuration.py
@@ -1,12 +1,14 @@
 def ttreecacheMB() : return 20
-def trace() : return True
+def trace() : return False #True
 def nCoresDefault() : return 4
-def useCachedFileLists() : return True
+def useCachedFileLists() : return False # True
 def maxArrayLength() : return 256
 def computeEntriesForReport() : return False
 def printNodesUsed() : return False
 def fakeString() : return ";FAKE"
-def cppFiles() : return ["cpp/linkdef.cxx"]
+def cppFiles() : return ["cpp/linkdef.cxx",
+                         "../../RootCore/scripts/load_packages.C"
+                         ]
 def whereami() : return max('/'.join(__file__.split('/')[:-1]), '.')
 def hadd() : return ['hadd',
                      whereami()+'/run/phaddy'][1]
